ext/afform/core/Civi/Api4/Afform.php

```diff
@@ -168,6 +168,13 @@ public static function getFields($checkPermissions = TRUE) {
           'name' => 'is_token',
           'data_type' => 'Boolean',
         ],
+        [
+          'name' => 'authx_timeout',
+          'data_type' => 'Integer',
+        ],
```

Comment:
We also need to enforce permissions at the api level. I think that because Afform is a nonstandard API this metadata won't automatically work, but it's still semantically correct to add it:
```suggestion
          'permission' => ['all CiviCRM permissions and ACLs'],
        ],
```
Then I guess we need to add some handling for this in the Afform::save api, e.g. loop through `getFields` and unset any values the user doesn't have permission to edit prior to saving (and prior to merging values with the existing form during update, so that the user isn't able to change an exiting value)